superset-frontend/src/features/alerts/AlertReportModal.tsx

```diff
@@ -1010,6 +1014,29 @@ const AlertReportModal: FunctionComponent<AlertReportModalProps> = ({
     return hasInfo;
   };
 
+  const checkEmailFormat = () => {
+    if (!notificationSettings.length) {
+      return true;
+    }
+
+    const validateEmails = (emails: string): boolean => {
+      if (!emails) return true; // No emails to validate
+      return emails
+        .split(/[,;]/)
```

Comment:
Should we add whitespace?
```suggestion
        .split(/[,\s;]/)
```

Reply:
Thanks @villebro for reviewing. The Alerts and Report modal for mail notifications accepts input as either comma (",") or semicolon (";") separated values. Therefore, whitespace parsing has been avoided.


<img width="350" alt="image" src="https://github.com/apache/superset/assets/117266407/7b6311b3-d4e6-4cea-abd6-e256eda3ac65">

Reply:
@villebro - Shall I resolve this comment ?. @kasiazjc Could you please help me here ?

Reply:
Thanks for clarifying - resolving the comment 👍